ttf2ugui.c

```diff
@@ -325,16 +317,16 @@ static UG_FONT *convertFont(const char *font, int dpi, float fontSize,int bitsPe
  * If DPI is not given, use pixes to specify the size.
  */
   if (dpi > 0)
-    error = FT_Set_Char_Size(face, 0, fontSize * 64 * bpp_mul, dpi, dpi);
+    error = FT_Set_Char_Size(face, 0, fontSize * 64, dpi, dpi);
   else
-    error = FT_Set_Pixel_Sizes(face, 0, fontSize * bpp_mul);
+    error = FT_Set_Pixel_Sizes(face, 0, fontSize);
   if (error) {
 
     fprintf(stderr, "set pixel sizes err %d\n", error);
     exit(1);
   }
 
-  int i, j,i_idx,j_idx;
```

Comment:
formatting: space after comma

Reply:
Those parts of the line were removed anyways